### ISSUE TITLE
automate testing, publishing and releases

### ISSUE THREAD

**Issue by matthewleon** [morganthomas/purescript-group#10]
Could model this on the way the core libs do it. Any objections?

**Comment by morganthomas** [comment]
Sounds good to me! I don't know the details of this proposal but I don't see how it could fail to be a good thing.

**Comment by hrb90** [comment]
Yeah, this sounds like a great idea. Any progress on it so far? I might be interested in taking a crack at it later this week, if not.

**Comment by matthewleon** [comment]
Go for it. You should mostly be able to copy configuration from core libs.
